rstream/exceptions.py

```diff
@@ -81,3 +81,6 @@ class PreconditionFailed(ServerError):
 
 class PublisherDoesNotExist(ServerError):
     code = 18
+
+class NoOffsetError(ServerError):
```

Comment:
nit: maybe `OffsetNotFound` would be a slightly better name? As in dotnet [client](https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/blob/main/RabbitMQ.Stream.Client/ResponseCode.cs#L27)